test/test_jit.py

```diff
@@ -12994,6 +12994,21 @@ def foo(a, b):
             return a, b
         self.checkScript(foo, (torch.rand(3), torch.rand(3)))
 
+    def test_ignored_props(self):
+        class A(nn.Module):
+            __jit_ignored_attributes__ = ["ignored"]
+
+            def __init__(self):
+                super().__init__()
+
+            @property
+            def ignored(self):
+                raise ValueError("shouldn't be called")
+
+        f = torch.jit.script(A())
```

Comment:
maybe let's use assertNoError context manager for this compilation?

Reply:
Don't think it exist?

Reply:
I see, let's keep it. Also, can you add the case that within a `torch.jit.ignore` function of the same module, after scripted, it can still access the property? would like to make sure ignored function can still access this kind of ignored attribute